source/reference/api/start.txt

```diff
@@ -273,86 +312,127 @@ Response
-------------------------------------
+Start a Reversible Sync Job
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. include:: /includes/intro-start-api-example-intro.rst
 
 The ``reversible`` and ``enableUserWriteBlocking`` fields allow the sync
 to be reversed. To reverse the sync direction, see: :ref:`reverse
 <c2c-api-reverse>`.
 
-Request
-~~~~~~~
+Request:
 
 .. literalinclude:: /includes/api/requests/start-reversible.sh
    :language: shell
 
-Response
-~~~~~~~~
+Response:
 
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
 
-Example: Start a Filtered Sync Job
-----------------------------------
+Start a Filtered Sync Job
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. include:: /includes/example-filter-collection.rst
 
 The ``includeNamespaces`` option creates a filter. To filter the sync,
 see: :ref:`c2c-filtered-sync`
 
-Request
-~~~~~~~
+Request:
 
 .. literalinclude:: /includes/api/requests/start-filtered.sh
    :language: shell
 
-Response
-~~~~~~~~
+Response:
 
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
-Example: Start Sync from Replica Set to Sharded Cluster
--------------------------------------------------------
+Start Sync from Replica Set to Sharded Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Request
-~~~~~~~
+Request:
 
 .. literalinclude:: /includes/api/requests/start-rs-shard.sh
 
-Response
-~~~~~~~~
+Response:
 
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
+
+.. _c2c-disabled-verifier:
+
+Start with the Verifier Disabled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, the embedded verifier runs by default when you
+start a migration. If you need to disable it, set
+``verification.enabled`` to ``false``.
+
+Request:
+
+.. literalinclude:: /includes/api/requests/start-no-verifier.sh
+
+Response:
+
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+You should verify that a migration was successful before
+transferring your application load to the destination cluster.
+If you need to disable the verifier for any reason, use an
+alternative :ref:`method <c2c-verification>` to verify sync.
+
 Behavior
 --------
 
+.. _c2c-start-verifier:
+
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` provides an embedded verifier to
+determine whether data transfer from the source cluster to the
+destination was successful.
+
+When enabled, the verifier performs a series of verification
+checks on the destination cluster. If any of these checks return
+an error, the verifier fails the migration. If all checks
+succeed, ``mongosync`` transitions to the ``COMMITTED`` state.
+
+To disable the verifier, set the ``verification.enabled``
+parameter in ``/start`` to ``false``. It's enabled by default. 
+
+The ``/start`` endpoint returns an error if you enable
+verification checks that are not supported by the source or
```

Comment:
Can we also add `or there is insufficient memory...`

Reply:
Fixed